### ISSUE TITLE
Jump servers UX improvements

### ISSUE THREAD

**Issue by E1izabeth** [dbeaver/dbeaver#34418]
### Description

1. The user was trying to double-click on table cells in order to edit the jump server host and other information. However, it was not obvious to the user that the text fields above the table had changed and that they should be writing information there. When the user tried to edit the host by double-clicking, they became confused and lost.

2. More a bug actually - last jump server host is not shown right in the table, until you reopen the wizard (video below)

https://github.com/dbeaver/dbeaver/assets/28875055/fc706899-fdb4-4d89-84b5-cbf83aeb0444

### DBeaver Version

DBeaver 24.1.0

**Comment by E1izabeth** [comment]
We want to fix the bug and left UI as is for now.

**Comment by Spongman** [comment]
yeah, this is really confusing. the fields in the table don't update live when you edit the values in the textboxes, and there's no obvious way to commit changes to the table.

what's worse is that this new table is in the same location as the old "Jump server settings",  and has a very similar name "Jump servers", so anyone used to the previous UI is going to think that they have similar functionality.

**Comment by E1izabeth** [comment]
Thank you for feedback, @Spongman

**Comment by davidv399** [comment]
Hi @E1izabeth 
Good Catch! Can I pick this one?

**Comment by E1izabeth** [comment]
Hi, @jack-stormentswe 
Yes, you could try
